truffe2/rights/tests.py

```diff
@@ -1,3 +1,6 @@
 from django.test import TestCase
 
-# Create your tests here.
+class RightsNoLoginTest(TestCase):
+    
+    def test_basic(self):
```

Comment:
I don't understand the purpose of this test, can you please add a comment?

Reply:
Test removed in "Step2"